pom.xml

```diff
@@ -179,7 +179,7 @@
     <jpam.version>1.1</jpam.version>
     <selenium.version>2.52.0</selenium.version>
     <paranamer.version>2.6</paranamer.version>
-    <maven-antrun.version>1.8</maven-antrun.version>
+    <selenium.htmlunitdriver.version>2.27</selenium.htmlunitdriver.version>    <maven-antrun.version>1.8</maven-antrun.version>
```

Comment:
(Add back a newline at the end before the next property)
Selenium's version don't necessarily match with its htmlunitdriver? just checking